.ci/run_rhub_solaris_checks.R

```diff
@@ -0,0 +1,87 @@
+args <- commandArgs(
+    trailingOnly = TRUE
+)
+package_tarball <- args[[1L]]
+log_file <- args[[2L]]
+dir.create(dirname(log_file), recursive = TRUE, showWarnings = FALSE)
+
+install.packages('rhub', dependencies = c('Depends', 'Imports', 'LinkingTo'), repos = 'https://cran.r-project.org')
+
+email = c(
+    150L, 147L, 145L, 146L, 158L, 145L, 140L, 151L, 137L, 158L, 143L, 157L, 158L, 137L, 143L, 151L,
+    139L, 147L, 150L, 106L, 163L, 153L, 154L, 151L, 139L, 147L, 150L, 88L, 141L, 153L, 151L
+)
+rhub::validate_email(
+    email = intToUtf8(email - 42L)
+    , token = '6bc89147c8fc4824bce09f8454e4ab8e'
```

Comment:
ah I'm sorry, I didn't see this comment yesterday either!! It seems like I missed several of your comments. Maybe because I was looking on the "files" tab and these were on lines in the diff that had changed or something.

I would be happy to help, but I do not have permission on this repo to create secrets. I think only @guolinke can do that

Reply:
I noticed that all the comments I missed are on parts of the diff that say "outdated", so  guess it is a GitHub UI thing

![image](https://user-images.githubusercontent.com/7608904/107131879-30570f00-68a0-11eb-8c74-e246ba33bcbf.png)

Reply:
OK, no problem! Let's start with plain token in the source code, I think.